src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -23,7 +24,22 @@ private ArrayList<Product> createProducts() {
          * one per each category in Category enum.
          * Product should not have any reviews.
          */
-        throw new UnsupportedOperationException("createProducts() is not implemented yet");
+        ArrayList<Product> myList = new ArrayList<>();
+        Product product1 = new Product("Harry Potter and the Philosopher's Stone", 15, Category.BOOKS);
```

Comment:
product1 and so on are not a good practice. again could add context like book, laptop, toy, food, etc